log/sockstatlog/logger.go

```diff
@@ -137,19 +155,19 @@ func (l *Logger) Write(p []byte) (int, error) {
 }
 
 // poll fetches the current socket stats at the configured time interval,
-// calculates the delta since the last poll, and logs any non-zero values.
+// calculates the delta since the last poll, and writes any non-zero values to the logger event channel.
```

Comment:
Nit: seems like this should wrap at ~80 characters, like the previous line break.